src/locale/en-US.json

```
@@ -97,6 +97,9 @@
       },
       "banner-message": {
         "text": "Banner message"
+      },
+      "logoUrl": {
```

Dans le fichiers de trads on a pris l'habitude d'utilise le kebab-case

```suggestion
      "logo-url": {
```